src/jit/compiler.h

```diff
@@ -7453,7 +7468,8 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
         bool compUseFCOMI;
         bool compUseCMOV;
 #ifdef _TARGET_XARCH_
-        bool compCanUseSSE2; // Allow CodeGen to use "movq XMM" instructions
+        bool compCanUseSSE2;   // Allow CodeGen to use "movq XMM" instructions
+        bool compCanUseSSE3_4; // Allow CodeGen to use SSE3, SSSE3, SSE4.1 and SSE4.2 instructions
 
```

Comment:
I also find this a bit confusing, but agree that it's probably best to keep in sync with the VM naming. However, FWIW I don't believe that Intel has ever guaranteed that they are cumulative. That said, the VM code checks for all the relevant flags before setting `CORJIT_FLAG_USE_SSE3_4`

---
In reply to: [89868211](https://github.com/dotnet/coreclr/pull/8291#discussion_r89868211) [](ancestors = 89868211,89865129)